functions/Get-DbaDbExtentDiff.ps1

```diff
@@ -167,6 +167,9 @@ function Get-DbaDbExtentDiff {
                     }
                 }
             }
+
+            # Close non-pooled connection as this is not done automatically. If it is a reused Server SMO, connection will be opened again automatically on next request.
+            $server.ConnectionContext.Disconnect()
```

Comment:
Should these be enclosed in if IsOpen or something?

Reply:
No, you can issue `$server.ConnectionContext.Disconnect()` as often as you want without a problem:
![image](https://user-images.githubusercontent.com/66946165/127546148-039c3d5b-3e71-4da8-9b24-19479d839c92.png)